util/mxs-sign-util.js

```diff
@@ -140,17 +140,17 @@ module.exports = {
       );
     }.bind(this);
 
-    if (!debug) {
-      signFile();
-      return;
-    }
-    let python = path.join(mlsdkRoot,
+    const python = path.join(mlsdkRoot,
```

Comment:
Figured these were meant to be `const` instead of `let` because they never change after instantiation.